agents/weather_agent/weather_monitor_agent.py

```diff
@@ -0,0 +1,189 @@
+
+import time
+import os
+import argparse
+import warnings
+import txaio
+
+from typing import Optional
+
+from weather_monitor import WeatherMonitor
```

Comment:
Move weather_monitor.py to `socs/socs/agents/weather_monitor/`, and use a package import instead of relative import. Something like: `from socs.agents.weather_monitor.weather_monitor import WeatherMonitor`